build.gradle

```diff
@@ -89,7 +89,8 @@ def versions = [
   logback         : '1.2.3',
   springBoot      : springBootVersion,
   jackson         : dependencyManagement.importedProperties['jackson.version'],
-  springfoxSwagger: '2.8.0'
+  springfoxSwagger: '2.8.0',
+  reformJavaLogging: '1.6.1'
```

Comment:
2.0.0 has been released perhaps update to it?

Reply:
🙉 it was still 1.6.1 this morning!